### ISSUE TITLE
Чтение дня показывает не все зачала

### ISSUE THREAD

**Issue by dmatora** [alexeyismirnov/prayerbook_ios7#2]
Если сравнить список зачал в воскресный день с http://days.pravoslavie.ru/Days/20170227.html, окажется что список не полный

**Comment by alexeyismirnov** [comment]
Да, такой вопрос часто возникает у пользователей. Однако, у меня никогда не было цели сделать полный календарь. Изначально, приложение задумывалось исключительно для мирских целей, а не для богослужебных. 

Дело в том, что я вообще противник использования телефонов на службах. А для мирян вполне хватит и одного чтения в день. Многие из нас читают Евангелие ежеднево? Немногие. Какой смысл перегружать людей списком из 10 чтений? 

Ну, в крайнем случае, можно сделать опцию - полный/неполный список чтений. Но это не приоритетно. Если что - присылайте Pull request :-)

**Comment by dmatora** [comment]
Ну я кстати ориентируюсь как раз на мирские цели, возможно пока недовник в что для чего.
